### DISCUSSION TITLE
Skin

### DISCUSSION THREAD

**Discussion by charleshys** [nftchef/art-engine#78] [Help + Q&A]
Hi there, may I know how to use flagging incompatible traits when it comes to skin?
Like I have skin with ears and without ears, some hats need to use skin without ears.

And at the same time still maintain its rarity distribution even there are two skin variations (with/without ears)

Thank you!

**Comment by nftchef** [comment]
The best way to handle something like this is with branching, through folder organization
https://generator.nftchef.dev/readme/branching-if-then
